src/Utility/userSupport.ts

```diff
@@ -10,7 +10,18 @@ export class UserSupport {
    */
   /* eslint-disable */
   public prepareRawProperties(rawProperties: string): string {
-    return '"' + rawProperties.replace(/"/g, '\\"') + '"';
+    return '"' + this.replaceSpecialCharacters(rawProperties).replace(/"/g, '\\"') + '"';
+  }
+
+  public replaceSpecialCharacters(rawProperties: string): string {
```

Comment:
Do we have a list of special characters which needs to be handled? Are there other such characters?

Reply:
Create a list of these characters in a constant file and iterate over that list to replace.

Reply:
characters are mentioned in https://provartesting.atlassian.net/browse/PDX-33
I'll create a list

Reply:
characters like '@' works fine, Priya has analysed the characters list.

Reply:
> characters are mentioned in https://provartesting.atlassian.net/browse/PDX-33 I'll create a list

Please confirm the same with Samuel. I think there might be more characters.

Reply:
As discussed in the call, all special characters are working except " and \

Reply:
But the list suggests otherwise. We have limited characters in the list, so come all the special characters are working?

Reply:
Yes, JSON library we are using is  Gson and it automatically handles standard characters and escape sequences except for few, all other characters that are not explicitly handled will still be processed normally using the library’s default behavior.

Reply:
either we can provide custom serializer and deserializer for our use.

Reply:
Please add a code comment to explain this behaviour